arcgis-ios-sdk-samples/Maps/Mobile map (search and route)/MobileMapViewController.swift

```diff
@@ -78,35 +78,35 @@ class MobileMapViewController: UIViewController, AGSGeoViewTouchDelegate {
         return markerSymbol
     }
     
-    private func labelSymbolForStop(text:String) -> AGSTextSymbol {
-        let symbol = AGSTextSymbol(text: text, color: UIColor.whiteColor(), size: 15, horizontalAlignment: .Center, verticalAlignment: .Middle)
+    fileprivate func labelSymbolForStop(_ text:String) -> AGSTextSymbol {
```

Comment:
similar to other comments above about having argument name to clarify call site usage